### PR TITLE
fix missing external-secrets definition

### PR DESCRIPTION
**Pull request by scoheb**

### CODE REVIEW

**Review by Michkov — APPROVED** [review]

File: ci/teams/rhtap-servicerelease/kustomization.yaml [hunk 1]
```diff
@@ -7,6 +7,7 @@ resources:
 - release-strategy.yaml
 - persistent-volume-claim.yaml
 - release-pipeline-sa.yaml
+- external-secrets
```
Comment (Michkov) [inline ci/teams/rhtap-servicerelease/kustomization.yaml:10]: https://github.com/redhat-appstudio/infra-deployments/blob/main/components/cluster-secret-store/base/appsre-stonesoup-vault-secret-store.yaml -> the namespace rhtap-servicerelease-tenant should be there

Comment (scoheb) [reply ci/teams/rhtap-servicerelease/kustomization.yaml:10]: fixed.

**Review by Michkov — APPROVED** [review]

**Pull request merged**